api-report/tree.api.md

```diff
@@ -6,20 +6,44 @@
 
 import { Serializable } from '@fluidframework/datastore-definitions';
 
+// @public
+export type Brand<ValueType, Name extends string> = ValueType & BrandedType<ValueType, Name>;
+
+// @public
+export abstract class BrandedType<ValueType, Name extends string> {
+    protected readonly _type_brand: Name;
+    // Warning: (ae-forgotten-export) The symbol "Invariant" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    protected _typeCheck?: Invariant<ValueType>;
+}
+
+// Warning: (ae-forgotten-export) The symbol "ExtractFromOpaque" needs to be exported by the entry point index.d.ts
```

Comment:
Need to export.

Reply:
Oops, I didn't realize thats not an error on CI, and can occur in the middle of the file. I only fixed the ones that were at the top! Fixed.